src/topp/ConsensusID.cpp

```diff
@@ -209,6 +209,22 @@ class TOPPConsensusID :
     StringList merged_spectra_data;
     String engine = prot_ids[0].getSearchEngine();
     String version = prot_ids[0].getSearchEngineVersion();
+
+    // merge proteins
+    unordered_set<String> seen_proteins;
+    vector<ProteinHit> merged_proteins;
+    for (Size i = 0; i < prot_ids.size(); ++i)
+    {
+      for (auto& hit : prot_ids[i].getHits())
+      {
+        const auto& iter_inserted = seen_proteins.emplace(hit.getAccession());
+        if (iter_inserted.second)
+        {
+          merged_proteins.emplace_back(std::move(hit));
```

Comment:
I think emplace_back is used if the object is constructed in-place.
push_back(T&&) also exists and might be more appropriate to use here.